src/smart-components/Channel/components/MessageInput/VoiceMessageInputWrapper.tsx

```diff
@@ -76,7 +76,11 @@ export const VoiceMessageInputWrapper = ({
   useEffect(() => {
     if (isSubmited && audioFile) {
       onSubmitClick(audioFile, recordingTime);
+      setSubmit(false);
+      setAudioFile(null);
     }
+  }, [isSubmited, audioFile, recordingTime]);
+  useEffect(() => {
     if (audioFile) {
       if (recordingTime < minRecordingTime) {
         setVoiceInputState(VoiceMessageInputStatus.READY_TO_RECORD);
```

Comment:
btw, dont we need this? `pause` https://github.com/sendbird/sendbird-uikit-react/pull/452/files#diff-71407a1f467cdb7fcb22c6a8748981f83c879b2797f92a7104a46fd00585986dR109

Reply:
Yes, it's for pausing the playing audio before sending message